documentation/asciidoc/computers/linux_kernel/building.adoc

```diff
@@ -104,79 +134,70 @@ $ make bcm2711_defconfig
 | Raspberry Pi 400
 | Raspberry Pi Compute Module 4S
 | Raspberry Pi Compute Module 4
+|===
 
+[NOTE]
+====
+The 32-bit distribution of Raspberry Pi OS on a Raspberry Pi 4B, 5, 400, Compute Module 4, or Compute Module 4S uses a 32-bit userland, but a _64-bit kernel_. To build a 32-bit kernel, set `ARCH=arm`. To boot a 32-bit kernel, set `arm_64bit=0` in `config.txt`.
+====
 
-.10+^.^| 64-bit
-| Raspberry Pi 3
-.9+.^|
-[source,console]
+[[native-customisation]]
+==== Customise the kernel version using `LOCALVERSION`
+
+Adjust `LOCALVERSION` to clarify that you are running your own kernel in the output of `uname` and prevent the kernel from overwriting existing modules in `/lib/modules`.
+
+To adjust `LOCALVERSION`, change the following line in `.config`:
+
+[source,ini]
 ----
-$ cd linux
-$ KERNEL=kernel8
-$ make bcm2711_defconfig
+CONFIG_LOCALVERSION="-v7l-MY_CUSTOM_KERNEL"
 ----
-| Raspberry Pi 3+
-| Raspberry Pi 4
-| Raspberry Pi 400
-| Raspberry Pi Zero 2 W
-| Raspberry Pi Compute Module 3
-| Raspberry Pi Compute Module 3+
-| Raspberry Pi Compute Module 4S
-| Raspberry Pi Compute Module 4
 
+You can also change this setting graphically with `menuconfig` at *General setup* > *Local version - append to kernel release*. For more information about `menuconfig`, see xref:linux_kernel.adoc#configure-the-kernel[the kernel configuration instructions].
+
+[[native-build]]
+==== Build
+
+Next, build the kernel. This step can take a long time, depending on the Raspberry Pi model.
+
+For 64-bit kernels, run the following commands to build a fresh kernel:
 
-| Raspberry Pi 5
-.1+.^|
 [source,console]
 ----
-$ cd linux
-$ KERNEL=kernel_2712
-$ make bcm2712_defconfig
+$ make -j6 Image.gz modules dtbs
+$ sudo make modules_install
```

Comment:
Perhaps it makes sense for the `modules_install` line to be in the "Install the kernel" section rather than the "Build" section? :thinking: